Directory out of date error

<!-- Thank you for reporting a possible bug.  Please fill in as much of the template below as you can. -->

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

I get a directory out of date using the action for some strange reason. I did look into the possible causes and this [Stackover Issue](https://stackoverflow.com/a/3298401/5956589) seems to address the issue.

**Additional context**
<!-- Add any other context about the problem here. -->

This might be an svn related issue refusing to commit files because of a date mismatch. using bash only, I resolved this by just running `svn update` before the commit. I have not added the steps to reproduce as it happens on a fresh commit, might be times from git. Have attached a snippet of the error message.

```
➤ Committing files...
Adding         tags/1.0.4
svn: E155011: Commit failed (details follow):
svn: E155011: Directory '/home/runner/svn-force-reinstall/tags/1.0.4' is out of date
svn: E175014: Precondition on '/!svn/rvr/2733523/force-reinstall/trunk' failed
Error: Process completed with exit code 1.
```

--- a/deploy.sh
+++ b/deploy.sh
@@ -147,6 +147,9 @@ if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -n
     svn propset svn:mime-type "image/svg+xml" "$SVN_DIR/assets/*.svg" || true
 fi
 
+#Resolves => SVN commit failed: Directory out of date
+svn update
+
 svn status
 
 echo "➤ Committing files..."

